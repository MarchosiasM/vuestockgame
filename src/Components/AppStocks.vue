<template>
  <div class="container">
    <div class="row">
      <div v-for="company in companies" :key="company.index">
        <app-stocks-card :name="company.name" :price="company.price">

        </app-stocks-card>
      </div>
    </div>
  </div>
</template>

<script>
import AppStocksCard from './AppStocksCard.vue';
export default {
  components: {
    AppStocksCard
  },
  data() {
    return {
      companies: this.$store.state.companies
    };
  }
};
</script>
