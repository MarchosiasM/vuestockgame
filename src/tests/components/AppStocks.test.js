import { mount } from '@vue/test-utils';
import AppStocks from '../../Components/AppStocks.vue';

describe('App Stocks', () => {
  it('renders correctly', () => {
    const wrapper = mount(AppStocks);
    expect(wrapper.element).toMatchSnapshot();
  });
});
