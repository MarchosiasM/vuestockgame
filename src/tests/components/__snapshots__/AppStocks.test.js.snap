// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Stocks renders correctly 1`] = `<div />`;
